extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/dashmanifestcreators/YoutubeDashManifestCreatorsUtils.java

```diff
@@ -566,10 +558,11 @@ public static Response getInitializationResponse(@Nonnull String baseStreamingUr
                                                      @Nonnull final ItagItem itagItem,
                                                      final DeliveryType deliveryType)
             throws CreationException {
-        final boolean isHtml5StreamingUrl = isWebStreamingUrl(baseStreamingUrl)
-                || isTvHtml5SimplyEmbeddedPlayerStreamingUrl(baseStreamingUrl);
-        final boolean isAndroidStreamingUrl = isAndroidStreamingUrl(baseStreamingUrl);
-        final boolean isIosStreamingUrl = isIosStreamingUrl(baseStreamingUrl);
+        final boolean isHtml5StreamingUrl = YoutubeParsingHelper.isWebStreamingUrl(baseStreamingUrl)
```

Comment:
Why are you changing imports? This is an unrelated change, and also related to personal preference, so it should be avoided. Also in other files. Please undo it.